gather_istio.sh

```diff
@@ -149,11 +149,23 @@ function getEnvoyConfigForPodsInNamespace() {
   done
 }
 
+function version() {
+  if [[ -n $OSSM_MUST_GATHER_VERSION ]] ; then
+    echo "${OSSM_MUST_GATHER_VERSION}"
+  else
+    echo "0.0.0-unknown"
+  fi
+}
+
 function main() {
   echo
   echo "Executing Istio gather script"
   echo
 
+  versionFile="${BASE_COLLECTION_PATH}/version"
+  echo "openshift-service-mesh/must-gather"> "$versionFile"
```

Comment:
is this string meant to resemble the container image name? what is this version file used for?

Reply:
> Red Hat Insights team needs a marker which is a unique file that identifies which data collections (Logging Must-Gather archive, OCS/ODF Must-Gather archive, etc) are handled.

Reply:
https://github.com/openshift/enhancements/blob/master/enhancements/oc/must-gather.md#must-gather-images, bullet 3:

> Must produce a text /must-gather/version that indicates the product (first line) and the version (second line, major.minor.micro-qualifier), so that programmatic analysis can be developed.

Reply:
From slack: 

> An example version file for performance-addon-operator data collection :
```
performance-addon-operator/must-gather
4.10 4.0-art-dev
```

Reply:
Thank you Marko. Sorry, should have checked the issue description

Reply:
Yeah, no worries. This Jira is a mess (with the links to different slack discussions and everything).